src/main/deltachat/settings.js

```diff
@@ -27,4 +27,16 @@ module.exports = class DCSettings extends SplitOut {
   keysExport (directory) {
     this._dc.importExport(C.DC_IMEX_EXPORT_SELF_KEYS, directory)
   }
+
```

Comment:
with this method we could update the settings in the database, but it won't update the prefixed keys (configured_mail_user etc)

Reply:
Prefixed keys are updated by the core, there is no need to ever update them manually. If they are not updated, that means configuration didn't succeed